animated_toolbox/css/scss/_navs.scss

```diff
@@ -0,0 +1,23 @@
+.primary-nav {
+    background-color: $black16;
+    overflow: hidden;
```

Comment:
@praxoc , you can remove the overflow hidden in the wrapper and float: left in the nav-item-link. Instead you can use display: flex in primary-nav